cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,69 @@
-describe('', () => {
-  before(() => {
+/// <reference types="cypress" />
 
-  });
+const { faker } = require('@faker-js/faker');
 
-  it('', () => {
+describe('article', () => {
+  beforeEach(() => {
+    cy.visit('user/register');
+  });
 
+  it('should create an article', () => {
+    const randomNumber = Math.random().toString().slice(2, 10);
+    const user = {
+      username: faker.lorem.word() + randomNumber,
+      email: faker.internet.email(),
+      password: faker.internet.password()
+    };
+    const article = {
```

Comment:
It'll be a better option to add a generateArticle() function to the config file and use it in the beforeEach() hook to set the variables.